component/service/src/main/java/org/exoplatform/social/rest/impl/user/UserRestResourcesV1.java

```diff
@@ -846,7 +876,7 @@ public Response updateUserById(@Context UriInfo uriInfo,
   @PATCH
   @Path("onboard/{id}")
   @Produces(MediaType.APPLICATION_JSON)
-  @RolesAllowed("administrators")
+  @RolesAllowed("users")
```

Comment:
you should check that the current user is admin or admin delegate.
Else a simple user could send onboarding mail to anybody